src/um/d2d1effects.rs

```diff
@@ -0,0 +1,644 @@
+// Copyright © 2016-2017 winapi-rs developers
```

Comment:
Whenever adding a new file, the start year for the copyright should be the year that file was added. Since you're adding this file in 2017, the start year should be 2017.